src/currency/crypto_currencies.rs

```diff
@@ -92,6 +92,26 @@ define_currency_set!(
             name: "ZCash",
             symbol: "ZEC",
             symbol_first: false,
+        },
+        // https://www.bitcoincash.org/
+        BCH: {
+            code: "BCH",
+            exponent: 8,            
+            locale: EnUs,
+            minor_units: 100_000_000,
+            name: "Bitcoin Cash",
+            symbol: "BCH",
+            symbol_first: false,
+        },
+        // https://bitcoinsv.com/
+        BSV: {
```

Comment:
thanks for adding these. 

i'm curious, what are you using that requires BSV support?